src/autocomplete/powershell.ts

```diff
@@ -167,6 +156,18 @@ ${flaghHashtables.join('\n')}
     return leafTpl
   }
 
+  private sanitizeSummary(summary?: string): string {
```

Comment:
why does this need to be a class method instead of a function?

Reply:
to avoid passing `this.config` on each call.

most ejs templates used in oclif assume `this.config` is available to use:
https://github.com/oclif/plugin-update/blob/4fd51bf5b661c69d246fb6f31e52d98d07de54be/src/commands/update.ts#L15-L30

so having it as a method makes it to pass `this.config` to `ejs` render func.